web-dapp/server-lib/logger.js

```diff
@@ -1,10 +1,30 @@
+'use strict';
+const winston = require('winston');
+
+const transports = [];
+
+if (process.env.NODE_ENV !== 'test') {
+    // Console transport
+    const consoleTransport = new winston.transports.Console({
+        level: process.env.LOG_LEVEL || 'debug',
+        colorize: true,
+        handleExceptions: true,
+        humanReadableUnhandledException: true
+    });
+    transports.push(consoleTransport);
+}
+
+const logger = new winston.Logger({
```

Comment:
also, is it possible to make `error`-level messages to write _both_ to `stderr` and `stdout`? it would make debugging easier (no need to watch 2 files simultaneously)

Reply:
Really, I don't know... but I'm going to look for and read about it